app/policies/orders_policy.rb

```diff
@@ -0,0 +1,15 @@
+class OrdersPolicy
+
+  def initialize(user, model)
+    @user = user
+    @model = model
+  end
+
+  def access_allowed?
+    !@user.nil? && @model.user_id == @user.id
```

Comment:
Everything except `false` and `nil` in Ruby is the truth. So you can write just:

 `@user && @user.id == @model.user_id`